File: Python/12250.py
# -*- coding: utf-8 -*-
"""
Coded on Spyder

@author: Iván
"""

greeting = str(input())
case = 0
while (greeting != '#'):
    case+=1
    if greeting == "HOLA":
        print("Case " + str(case) + ": SPANISH")
    elif greeting == "HALLO":
        print("Case " + str(case) + ": GERMAN")
    elif greeting == "HELLO":
        print("Case " + str(case) + ": ENGLISH")
    elif greeting == "CIAO":
        print("Case " + str(case) + ": ITALIAN")
    elif greeting == "BONJOUR":
        print("Case " + str(case) + ": FRENCH")
    elif greeting == "ZDRAVSTVUJTE":
        print("Case " + str(case) + ": RUSSIAN")
    else:
        print("Case " + str(case) + ": UNKNOWN")
        
    
    greeting = str(input())

File: C++/12250.cpp
#include <iostream>

// 12250 - Language Detection - Iván - Visual Code Studio

using namespace std;

int main(){
    
    string s = "";
    cin >> s;
    int i = 1;
    while (s != "#") {
        cout << "Case " << i << ": ";
        if (s == "HELLO"){
            cout << "ENGLISH" << endl;
        } else if (s == "HOLA"){
            cout << "SPANISH" << endl;
        } else if (s == "HALLO"){
            cout << "GERMAN" << endl;
        } else if (s == "BONJOUR"){
            cout << "FRENCH" << endl;
        } else if (s == "CIAO"){
            cout << "ITALIAN" << endl;
        } else if (s == "ZDRAVSTVUJTE"){
            cout << "RUSSIAN" << endl;
        } else {
            cout << "UNKNOWN" << endl;
        }
        cin >> s;
        i++;
    }
    return 0;
}
